### ISSUE TITLE
Ability to skip generation of html report

### ISSUE THREAD

**Issue by vmassol** [siom79/japicmp#121]
I'd like to skip generating the html file since I'm only using japicmp to fail the build when source/binary backward incompatibilty is found. Right now whenever a maven module is built japicmp generates 3 files:

`````
[INFO] Written file '/Users/vmassol/dev/xwiki/git/xwiki-platform/xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-ui/target/japicmp/japicmp.diff'.
[INFO] Written file '/Users/vmassol/dev/xwiki/git/xwiki-platform/xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-ui/target/japicmp/japicmp.xml'.
[INFO] Written file '/Users/vmassol/dev/xwiki/git/xwiki-platform/xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-ui/target/japicmp/japicmp.html'.
`````

Would it be possible to skip the generation of the html one at least?

The goal is to speed up the build.

Thanks

**Comment by siom79** [comment]
Added new options `skipHtmlReport`, `skipXmlReport` and `skipDiffReport` to skipt the corresponding report.

If you like, you can try it on the `develop` branch:
```
git clone https://github.com/siom79/japicmp.git
cd japicmp
git checkout develop
mvn install
```

**Comment by siom79** [comment]
Released with 0.7.2.
